src/org/centos/contra/Infra/Utils.groovy

```diff
@@ -481,16 +519,19 @@ def parseDistilledContext(String context_filename="linchpin.distilled") {
  * @param context
  * @return
  */
-static def getInstanceFromContext(instance, context) {
-    def instance_context = [:]
+def getInstanceFromContext(instance, context) {
+    def instanceContext = [:]
     context.each { k,v ->
-        String name = instance instanceof Openstack ? "${instance.getNameWithUUID()}1" : instance.getName()
```

Comment:
So this 1 at the end was required for the version of Linchpin, iirc. Have you verified that removing this 1 works properly?

Reply:
I'm not sure which version of linchpin appended a one, but I found that the new version appended a random UUID instead.

So to cover both use cases I compare the two strings with `startsWith` so that it is only matching the name which should still be unique. It was working with my OpenStack setup.

Reply:
Ok; did you try with multiple openstack instances by chance?

Reply:
I did not - but I will make a branch to test this use case.

Reply:
Is this the kind of thing you were looking for, or were you looking for multiple openstack instances in the same branch?
https://github.com/jaypoulz/multiarch-ci-test-template/blob/dev-v2.0.0-openstack/job.yml

Reply:
Results look good for the latest test. I ended up doing multiple openstack instances in one parallel thread and a single instance in another and the results look clean.
https://multiarch-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/multiarch-qe/job/jaypoulz/job/multiarch-ci-hdsl-test-template/job/dev-v2.0.0-openstack/